ambry-api/src/main/java/com/github/ambry/config/MySqlNamedBlobDbConfig.java

```diff
@@ -93,6 +102,8 @@ public class MySqlNamedBlobDbConfig {
   public final TransactionIsolationLevel transactionIsolationLevel;
 
   public MySqlNamedBlobDbConfig(VerifiableProperties verifiableProperties) {
+    this.listNamedBlobsSQLOption = verifiableProperties.getInt(LIST_NAMED_BLOBS_SQL_OPTION,
```

Comment:
If we only have two option, I would suggest to use boolean instead of switch.
The reason are:
1. for switch we'd better provide the default which throw IllegalArgumentException exception for invalid value.
2. for the config setting, we'd better provide the min and max value to avoid people put some number out of limit.
3. we will always need to check the code and understand which value we want to pick, for easy maintainance, probably a boolean value is more straightforward given that we only have two options.

Reply:
if my query works in prod, i plan to remove this config completely.

Reply:
sorry to be nitpicking, I just feel like even though we will remove it later, we'd better to always provide the better quality code cause sometimes you might be pull into other urgent task and this code will leave for a long time. That's just my preference. I would suggest we can at least add the min and max value for the config so if we forget to remove for a long time, we still have some constraint instead of return null only.